packages/expo-firebase-analytics/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "expo-firebase-analytics",
+  "version": "1.0.0",
```

Comment:
This is going to be a little confusing (we've got `expo-firebase-analytics@1.0.0-rc.*` and `@2.0.0-rc*`… Even more so since `build.gradle` defines version as `2.0.0`. Would it be possible to unify all the versions?